### PR TITLE
Optimizations to AppTP Manage Trackers View

### PR DESCRIPTION
**Pull request by SlayterDev**
Task/Issue URL: https://app.asana.com/0/1203881841413756/1204456266799291/f
Tech Design URL:
CC: @samsymons 

**Description**:
This PR adds optimizations to the Manage Trackers view to improve user experience when loading the view. Also optimizes toggling a tracker to not have to re-build the full tracker list.

**Steps to test this PR**:
1. Go to AppTP -> Manage Trackers. Previously there was a pause before the view opened. View should now open immediately and a brief spinner will show as the tracker list is built.
2. Toggle some trackers. Ensure they persist between view loads. Ensure trackers aren't blocked if they're allowed.

**Device Testing**:

* [ ] iPhone SE (1st Gen)
* [ ] iPhone 8
* [ ] iPhone X
* [x] iPhone 14 Pro
* [ ] iPad

**OS Testing**:

* [ ] iOS 14
* [ ] iOS 15
* [x] iOS 16

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

File: DuckDuckGo/AppTPManageTrackersView.swift [hunk 1]
```diff
@@ -29,49 +29,65 @@ struct AppTPManageTrackersView: View {
         viewModel.changeState(for: domain, blocking: isBlocking)
     }
     
+    var loadingStare: some View {
```
Comment (samsymons) [inline DuckDuckGo/AppTPManageTrackersView.swift]: Typo here:

```suggestion
    var loadingState: some View {
```

**Review by samsymons — APPROVED** [review]
Working well for me - just one small comment, then this is good to go.

**Comment by samsymons** [comment]
Actually, one other thing: I noticed after the fact that we have a lot of logs like this when browsing the allowlist, are we able to clean that up in this PR?

`No color named 'LightGreenBlue' found in asset catalog for main bundle`

**Comment by SlayterDev** [comment]
> Actually, one other thing: I noticed after the fact that we have a lot of logs like this when browsing the allowlist, are we able to clean that up in this PR?
> 
> 
> 
> `No color named 'LightGreenBlue' found in asset catalog for main bundle`

Yes 🙄 I swear I've made this change like twice. I'll try to remove the color and re-add it.

File: DuckDuckGo/AppTPManageTrackersView.swift (hunk 1 above)
Comment (SlayterDev) [reply DuckDuckGo/AppTPManageTrackersView.swift]: 😅

**Pull request merged**